sample/Kmd.Logic.Cpr.Client.Sample/Program.cs

```diff
@@ -50,6 +50,12 @@ private static void InitLogger()
 
         private static async Task Run(AppConfiguration configuration)
         {
+            var cprPersonId = Guid.Parse("5e3d9df6-d082-467a-88bd-dca56edc7328");
+            var datefrom = DateTime.Today.AddMonths(-2);
+            var dateTo = DateTime.Today;
+            var pageNo = 1;
+            var pageSize = 1;
```

Comment:
Maybe set pageSize to 10 or 100?

Reply:
Do we need these settings here at all? Can't we just put them directly in the request?

Reply:
Passing those as parameters of method directly